src/lib/Select/Select.svelte

```diff
@@ -6,14 +6,16 @@
 
   let selectedElementDiv: HTMLDivElement | null = null;
 
+  export let alt='';
```

Comment:
Can we have this `alt` text in `SelectProperties`?

Reply:
we can take alt directly as component prop